doc/index.rst

```diff
@@ -33,6 +33,7 @@
    orchestrator
    launchers
    ml_features
+   sr_logger
```

Comment:
I think this might be meant to be `ss_logger`

Reply:
I was wondering why it wasnt showing up